rs/sns/testing/src/main.rs

```diff
@@ -25,28 +78,110 @@ async fn main() {
         .await;
     let endpoint = pocket_ic.make_live(Some(8080)).await;
     println!("PocketIC endpoint: {}", endpoint);
-    bootstrap_nns(&pocket_ic).await;
-    let greeting = "Hello there".to_string();
-    let test_canister_id = install_test_canister(
+
+    let registry_proto_path = opts.state_dir.join("registry.proto");
+    let initial_mutations = load_registry_mutations(registry_proto_path);
+    let dev_principal_id = get_identity_principal(&opts.dev_identity).unwrap();
+
+    bootstrap_nns(
         &pocket_ic,
-        TestCanisterInitArgs {
-            greeting: Some(greeting),
-        },
+        vec![initial_mutations],
+        vec![
+            (
+                (*TREASURY_PRINCIPAL_ID).into(),
+                Tokens::from_tokens(10_000_000).unwrap(),
+            ),
+            (dev_principal_id.into(), Tokens::from_tokens(100).unwrap()),
+        ],
+        vec![dev_principal_id],
+    )
+    .await;
+}
+
+async fn run_basic_scenario(opts: RunBasicScenarioOpts) {
+    let dfx_interface = dfx_interface(&opts.network, Some(opts.dev_identity))
+        .await
+        .unwrap();
+
+    let network = dfx_interface.network_descriptor();
+
+    let dev_agent = dfx_interface.agent();
+    let treasury_agent = &build_ephemeral_agent(TREASURY_SECRET_KEY.clone(), &network.clone())
```

Comment:
can this be moved closer to its uage?  It appears to be unneeded up here pre-validation

Reply:
It does make sense, and this change was already done, but in the subsequent PR, that will be the part of the last milestone: https://github.com/serokell/ic/pull/7

Do you want me to port related changes to this PR?

Reply:
If it's done later, that's fine with me, feel free to resolve this.